content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -94,9 +100,9 @@ Customer Invoices / Vendor Bills`, and click on :guilabel:`⋮` and select :guil
 Matching`, or by going to :menuselection:`Accounting --> Reconciliation`.
 
 .. note::
-   During the reconciliation, if the sum of the debits and credits does not match, it means there is
-   a remaining balance that either needs to be reconciled at a later date or needs to be written off
-   directly.
+   During the :doc:`reconciliation <../../bank/reconciliation/use_cases>`, if the sum of the debits
+   and credits does not match, it means there is a remaining balance that either needs to be
+   reconciled at a later date or needs to be written off directly.
```

Comment:
During the :doc:`reconciliation <../../bank/reconciliation/use_cases>`, if the sum of the debits
   and credits does not match, there is a remaining balance. This either needs to be
   reconciled at a later date or needs to be written off directly.